nuxeo/templates/deployment.yaml

```diff
@@ -198,6 +198,16 @@ spec:
             subPath: nuxeo-{{ $key }}.conf
           {{- end }}
           {{- end }}
+          {{- with .Values.customContributions }}
+          {{- range $key, $val := . }}
+          - name: nuxeo-contributions
+            mountPath: /etc/nuxeo/contrib.d/{{ $key }}
+            subPath: {{ $key }}
+          {{- end }}
+          - name: nuxeo-init-script
```

Comment:
Probably doesn't matter, but should we have a different names for this volumeMount item than the one below?
Maybe: `nuxeo-init-script-contributions` and `nuxeo-init-script-db-template`.

Reply:
And yes to mount a single Configmap for contributions.